src/libs/actions/Receipt.js

```diff
@@ -0,0 +1,21 @@
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../ONYXKEYS';
+
+function onUploadReceiptError(isAttachmentInvalid, attachmentInvalidReasonTitle, attachmentInvalidReason) {
```

Comment:
3 variables seems too much, can we just use `errors`?

`isAttachmentInvalid` can be deducted from the `errors` if it's empty or not.

Reply:
Disagree, we need three in order to tell which type of error (too big, too small, wrong file type)